webui/react/src/themes.ts

```diff
@@ -23,11 +23,13 @@ import {
  * Where did we get our sizes from?
  * https://www.quora.com/What-is-the-difference-among-big-large-huge-enormous-and-giant
  */
-export enum ShirtSize {
-  small = 'small',
-  medium = 'medium',
-  large = 'large',
-}
+export const ShirtSize = {
+  small: 'small',
```

Comment:
This and a couple of other enums don't follow our `TitleCase` naming convention, would you mind fixing that?

Reply:
sure